src/components/NoLocalOrganisations/NoLocalOrganisations.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+import styles from './NoLocalOrganisations.module.scss';
+import HandHeart from '@/assets/logo/HandHeart';
+
+interface NoLocalOrganisationsProps {
+  organisationName?: string;
+}
+
+const NoLocalOrganisations: FC<NoLocalOrganisationsProps> = ({
+  organisationName = 'charities',
+}: NoLocalOrganisationsProps) => {
+  return (
+    <div className={styles.container}>
+      <h3>No {organisationName} have registered within your search area</h3>
+      <p>
+        <span className={styles.contactUs}>
+          <a href="/contact">Contact us</a>
```

Comment:
- Should use <Link> component from react router. 
- And use the link from the Paths global variable.

Reply:
thanks, this is now fixed